dash-spv-coinjoin/src/coinjoin_client_queue_manager.rs

```diff
@@ -48,23 +49,23 @@ impl CoinJoinClientQueueManager {
     }
 
     pub fn check_queue(&mut self) {
-        let current_time = Instant::now().elapsed().as_secs();
-        self.coinjoin_queue.retain(|q| !q.is_time_out_of_bounds(current_time as i64));
+        let current_time = SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_secs();
+        self.coinjoin_queue.retain(|q| !q.is_time_out_of_bounds(current_time));
```

Comment:
I looks like `Instant::now()` doesn't report correct time

Reply:
yes, it's for relative time measurement